src/lambda_codebase/initial_commit/bootstrap_repository/deployment/lambda_codebase/initial_commit/pipelines_repository/adf-build/deployment_map.py

```diff
@@ -20,8 +21,10 @@ def __init__(
             map_path=None
     ):
         self.map_path = map_path or 'deployment_map.yml'
+        self.map_dir_path = map_path or 'deployment_map'
```

Comment:
what if we called the folder "maps" or "deployment_maps" since it will probably hold multiple